apps/settings/js/simcard_fdn.js

```diff
@@ -2,9 +2,6 @@ define('simcard_fdn', ['modules/dialog_service'], function(DialogService) {
   'use strict';
 
   var SimFdnLock = {
-    dialog: document.getElementById('call-pin2-dialog'),
```

Comment:
Please also remove `call_pin2_dialog.html` and the corresponding element in `index.html`.